wiki/mapping/README.md

```diff
@@ -8,25 +8,21 @@ There is an active project underway to expand this section of the BSMG wiki.
 Visit `#mapping-discussion` on the BSMG Discord to get involved!
 :::
 
-**Many thanks to contributors from across the mapping community who have made this expanded wiki possible!**
+Much of the contents of 30+ written mapping guides and tutorials has been or is in the process of being added to this wiki so that it can be maintained by the community as a whole. Many many **MANY** thanks to the mappers who blazed this trail and contributed content including **Awfulnaut, BennyDaBeast, Checkthepan, Cyan Snow, Freeek, The Good Boi, Helen Carnate, Hexagonial, Kolezan, LittleAsi, Megalon, MandyNasty, Nik, Puds, Ris, Ryger, Scrappy, Silent Caay, Skyler Wallace, Sykes,** and **Uninstaller.**
```

Comment:
This is really minor, but I think this first sentence on the page here is difficult to read.
Maybe it could be simplified to just 
`Over *thirty* different mapping guides and tutorials have been combined into this wiki! ...`